test/storage/sql_table_test.cpp

```diff
@@ -791,4 +832,172 @@ TEST_F(SqlTableTests, MultipleColumnWidths) {
   txn_manager_.Commit(txn, TestCallbacks::EmptyCallback, nullptr);
   delete txn;
 }
+
+// NOLINTNEXTLINE
+TEST_F(SqlTableTests, BasicDefaultValuesTest) {
+  // TODO(Sai): Merge the sql_table_test and sql_table_concurrent_test frameworks to avoid repetition for adding in
+  // columns. Have 3 types of columns, 1 with default value at creation, one without and one with default value added
```

Comment:
Does this comment need to be updated?  I think you changed the test based off of our conversation (SqlTable wouldn't see a default value change that occurred after column creation), but this says you're testing it.

Reply:
Yes, it was outdated.